src/ngraph/runtime/cpu/builder/avg_pool.cpp

```diff
@@ -55,6 +55,10 @@ namespace ngraph
                     auto avg_pool_desc =
                         mkldnn_emitter->get_avg_pooling_forward_desc<ngraph::op::AvgPool>(node,
                                                                                           false);
+#if MKLDNN_VERSION_MAJOR >= 1
+                    mkldnn_emitter->query_scratchpad_pooling_forward(avg_pool_desc);
```

Comment:
Instead we can define a MACRO/function to do this. If `MKLDNN_VERSION_MAJOR < 1` it resolves/does nothing. Same for all other uses with #if

Reply:
Changed to macro.